CITATIONS.bib

```diff
@@ -0,0 +1,48 @@
+@article{dealII94,
+  title     = {The \texttt{deal.II} Library, Version 9.4},
+  author    = {Daniel Arndt and Wolfgang Bangerth and Marco Feder and
+               Marc Fehling and Rene Gassm{\"o}ller and Timo Heister
+               and Luca Heltai and Martin Kronbichler and
+	       Matthias Maier and Peter Munch and Jean-Paul Pelteret
+	       and Simon Sticko and Bruno Turcksin and David Wells},
+  journal   = {Journal of Numerical Mathematics},
+  doi       = {10.1515/jnma-2022-0054},
+  year      = {2022},
+  volume    = {30},
+  number    = {3},
+  pages     = {231-246},
+  url       = {https://dealii.org/deal94-preprint.pdf}
+}
+
+@article{dealII93,
```

Comment:
I guess we can drop this one!?

Reply:
I'm just mirroring https://www.dealii.org/publications.html and think we should be consistent. I slightly tend to remove the 9.3 release form both places.

Reply:
I see.

> I slightly tend to remove the 9.3 release form both places.

I agree!